src/routes/notification.route.js

```diff
@@ -10,10 +11,11 @@ const { checkNotificationExists, checkNotificationOwner } = NotificationMiddlewa
 const {
   getUserNotifications, updateNotificationStatus, getSingleNotification, markAllNotificationsAsRead
 } = NotificationController;
+const { idValidate } = Validation;
```

Comment:
great work! just set the name to this function to validateNotificationId it would be great